src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;1,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+
+    <div class="card" data-qa="card">
+      <div class="card__image">
+        <img 
+          class="card__image-imac"
+          src="images/imac.jpeg"
+          alt="Моноблок APPLE A1419 iMac 27 Retina 5K (MNED2UA/A)"
+        >
+      </div>
+      <main class="main">
+        <div class="main__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
```

Comment:
use h2

Reply:
also, the font-weight is different from the design

<img width="320" alt="Screenshot 2022-06-02 at 17 34 38" src="https://user-images.githubusercontent.com/57598825/171653936-34bfcbcf-9e5c-4dcc-bd08-5e22bf1040b0.png">